pages/api/itemDefinitions/index.ts

```diff
@@ -17,19 +19,23 @@ export default async function handler(
     switch (req.method) {
       case 'GET': {
         const items = await getItemDefinitions()
-
-        return res.status(200).json({
+        const resStatus = items.length ? 200 : 204
+        return res.status(resStatus).json({
           success: true,
           payload: items,
         })
       }
       case 'POST': {
+        apiItemDefinitionValidation(req.body)
         const itemDefinition = req.body as ItemDefinition
-        await createItemDefinition(itemDefinition)
+        let response = await MongoDriver.createEntity(
```

Comment:
use `const` here, the value isnt changed after assigning.

Reply:
fixed